migration/20160516-remove-work-was-merged-into.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM works where was_merged_into_id is not null;
+ALTER TABLE works DROP COLUMN was_merged_into_id CASCADE;
```

Comment:
Recreate the new views by copying your new versions of materialized_view_works.sql and materialized_view_works_workgenres.sql to the bottom of the migration script.